i18n/zh-CN/docusaurus-plugin-content-blog/2023-12-18-cloud-native-configuration-language-kcl/index.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: 2023-12-18-cloud-native-configuration-language-kcl
+title: KCD 社区会议回顾 | 云原生策略配置语言 KCL
+authors:
+  name: KCL Team
+  title: KCL Team
+tags: [KCL, Meeting]
+---
+
+> KusionStack 负责人李大元和云原生策略配置语言 KCL 项目的 Maintainer 宗喆在 2023 年 CNCF KCD 深圳会议的内容回顾，本文章主要是 KCL 语言部分演讲的内容回顾，活动链接: [https://community.cncf.io/events/details/cncf-kcd-hangzhou-presents-kcd-shenzhen-2023/](https://community.cncf.io/events/details/cncf-kcd-shenzhen-presents-kcd-shenzhen-2023/)
+
+## 自我介绍
+
+大家好，来自蚂蚁集团的宗喆，也是 KCL 语言项目的 Manitinor，很荣幸能够参加这次 KCD 深圳大会，接下来由我为大家介绍一些关于 KCL 语言部分的内容。
+
+![intro-myself.png](/img/blog/2023-12-18-cloud-native-configuration-language-kcl/intro-myself.png)
```

Comment:
This pic doesn't seem to be your self introduction.

Reply:
fixed